seerep_msgs/core/query.h

```diff
@@ -15,9 +14,8 @@ struct Query
 {
   Header header;
   std::optional<std::vector<boost::uuids::uuid>> projects;  ///< search all projects if not set
-  std::optional<AABB> boundingbox;
-  std::optional<Polygon2D> polygon;          // query point clouds in the region defined by this polygon
-  std::optional<Timeinterval> timeinterval;  ///< only do temporal query if set
+  std::optional<Polygon2D> polygon;                         // query point clouds in the region defined by this polygon
```

Comment:
`query point clouds` can you make this a bit more general? We also query other data types